src/apps/vpn/controller.cpp

```diff
@@ -55,6 +56,9 @@ constexpr const int CONNECTION_MAX_RETRY = 9;
 constexpr const uint32_t CONFIRMING_TIMOUT_SEC = 10;
 constexpr const uint32_t HANDSHAKE_TIMEOUT_SEC = 15;
 
+constexpr const uint32_t VPNSESSION_PING_TIMER_SEC = 10800;
+constexpr const uint32_t VPNSESSION_PING_TIMER_DEBUG_SEC = 10;
```

Comment:
This is kinda excessive even for debugging IMO. To be honest, I am not really a big fan of having a different timer interval for debugging, unless you are explicitly debugging this ping it's just more noise in the logs. If you are explicitly debugging this ping you can just change the timer yourself for testing purposes. **tl;dr; What if we just remove this debugging interval?**

Reply:
It was specifically written in the tech spec and I could see this potentially being useful for QA testing. @mcleinman thoughts?

Also, the debug timer is actually supposed to be 120 seconds - I had it set to 10 for... ultra debugging purposes (but this kind of proves your point 😂)

Reply:
LOL "ultra debugging". Ok, if it's in the spec just leave it in. We can always file a ticket to remove it later if it becomes annyoing.